### PR TITLE
Add script to generate winrt headers using cppwinrt.exe

### PR DESCRIPTION
**Pull request by clarkezone**
Generate winrt headers using the cppwinrt tool.  A subsequent engine PR will consume them.

https://github.com/flutter/flutter/issues/70205

Consumed by PR:
https://github.com/flutter/engine/pull/24861

### CODE REVIEW

File: build/win/generate_winrt_headers.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  """
+  
+  cppwinrt_exe = os.path.join(
+  __file__,
+  '..\\..\\third_party\\cppwinrt\\bin\\cppwinrt.exe')
+
+  args = [cppwinrt_exe, '-in', 'sdk',
+      '-out', '%s' % output_dir]
+
+  return 0
```
Comment (cbracken) [inline build/win/generate_winrt_headers.py:27]: Looks like we're not currently running the command here. Does this need a re-push?

**Review by cbracken — CHANGES_REQUESTED** [review]

File: build/win/generate_winrt_headers.py (hunk 1 above)
Comment (cbracken) [reply build/win/generate_winrt_headers.py:27]: Might also be a good idea for us to first recursively delete the output directory (if it exists) then run the command.

Comment (clarkezone) [reply build/win/generate_winrt_headers.py:27]: Good catch ;-)  I think i fat fingered the porting from my test branch.  On the delete front, I have the Clean() method that is called before `generate_headers()` but that isn't do a recursive.  I'll take another look to make sure that's working as expected

Comment (clarkezone) [reply build/win/generate_winrt_headers.py:31]: Both fixed in most recent commit

**Review by chinmaygarde — APPROVED** [review]

**Review by cbracken — COMMENTED** [review]
Thanks!

**Pull request merged**